quadrat/sass/theme.scss

```diff
@@ -10,3 +10,7 @@
 .wp-block-post-featured-image {
 	margin-top: 0;
 }
+
+.subscription-column {
+	text-align: right;
+}
```

Comment:
should this be inside [this](https://github.com/Automattic/themes/tree/make/quadrat/quadrat/sass/block-patterns) folder?